pypeit/pypeit.py

```diff
@@ -556,16 +536,38 @@ def extract_one(self, frames, det, bg_frames=[], std_outfile=None):
         # Object finding, first pass on frame without sky subtraction
         self.maskslits = self.caliBrate.maskslits.copy()
 
-        self.redux = reduce.instantiate_me(self.spectrograph, self.caliBrate.tslits_dict, self.mask, self.par,
+        self.redux = reduce.instantiate_me(self.spectrograph, self.caliBrate.tslits_dict,
+                                           self.mask, self.par,
                                            ir_redux = self.ir_redux,
                                            objtype=self.objtype, setup=self.setup,
                                            det=det, binning=self.binning)
 
+        # Prep for manual extraction
+        if self.par['scienceimage']['manual'] is not None:
+            # This requires sci_files
```

Comment:
I think you need to just use the first one if they are combining them, so this should be sci_files[0] or something.

Reply:
yes, this is what is done.